docs/azure-saas-docs/content/en/resources/container-publishing.md

```diff
@@ -0,0 +1,62 @@
+---
+type: docs
+title: "Container Publishing"
+weight: 300
+---
+
```

Comment:
This looks good for now, but I think we'll want to revisit some of the wording when you're finished with the current container publishing rework.

Reply:
This has been updated with new container publishing info.